receiver/kafkareceiver/README.md

```diff
@@ -28,6 +28,7 @@ The following settings can be optionally configured:
   - `zipkin_json`: the payload is deserialized into a list of Zipkin V2 JSON spans.
   - `zipkin_thrift`: the payload is deserialized into a list of Zipkin Thrift spans.
   - `raw`: (logs only) the payload's bytes are inserted as the body of a log record.
+  - `text`: (logs only) the payload are decoded as text and inserted as the body of a log record. By default, it uses UTF-8 to decode. You can use `text_<ENCODING>`, like `text_utf-8`, `text_shift_jis`, etc., to customize this behavior.
```

Comment:
How is this different from raw?

Reply:
> How is this different from raw?

- this sets `ObservedTime`
- this decodes the payload with different text encodings, and sets `Body` as `StringValue` instead of `BytesValue`